README.md

```diff
@@ -157,6 +157,11 @@ docker exec -it <the container name goes here> psql -U cratesfyi
 docker-compose run -p 3000:3000 web daemon --foreground
 ```
 
+### Changing the build environment
+
+To make a change to https://github.com/rust-lang/crates-build-env
```

Comment:
Maybe we want to have nice link texts here? Something like "To make a change to [the build environment] and test that it works on docs.rs, see [the wiki page]"

Reply:
Added this in the next commit.